projects/ZAP/README.md

```diff
@@ -0,0 +1,15 @@
+# ZAP Swag
+Artwork for all official ZAP swag - posters, stickers, t-shirts etc.
+
+All ZAP images are released under the Creative Commons license.
+
+You can also purchase ZAP merchandise online - we make no money out of these sales, if you'd like to donate money to the OWASP ZAP project them please use the 'donate' button on https://www.owasp.org/index.php/ZAP
+
+You can of course use the artwork from this repo with any other online store that you like.
+
+## Stores
+A range of products can be purchased from http://www.redbubble.com/people/zaproxy
+
+Stickers can be purchased from https://www.stickermule.com/uk/user/1070684077/stickers
```

Comment:
Stickermule user shops do not exist any more

Reply:
Oh yeah :/ I'll remove that...

Reply:
Arg, and I left the ZAP dir in there too :(